File: HK4/CNPM/Lab/Lab_new_2023-2024/Lab08-UnitTest/UnitTestExcercise/UnitTestExcercise/UnitTestExcercise/Student.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace UnitTest
{
    public class Student
    {
        public String ID { set; get; }
        public String Name { set; get; }
        public int Age { set; get; }

        public double Score { set; get; }

        public Student(String id, String name, int age, double score)
        {
            this.ID = id;
            this.Name = name;
            this.Age = age;
            this.Score = score;
        }

    }
}

File: Semester 2/Object Oriented Programming/LAB/Lab10/522H0090_VoNhatHao_Lab10/Bai1/Student.java
public class Student
{
    private String name;
    private String address;
    private String sex;
    private double score;

    public Student()
    {
        this.name = "";
        this.address = "";
        this.sex = "";
        this.score = 0.0;
    }

    public Student(String name, String address, String sex, double score)
    {
        this.name = name;
        this.address = address;
        this.sex = sex;
        this.score = score;
    }

    public class StudentOperater
    {
        public void print()
        {
            System.out.println("[" + name + ", " + address + ", " + sex + ", " + score + "]");
        }

        public String type()
        {
            if(score > 8)
            {
                return "A";
            }
            else if (score >= 5 && score <= 8)
            {
                return "B";
            }
            else
            {
                return "C";
            }
        }
    }
}